List-of-Free-Code-Camp-city-based-Campsites.md

```diff
@@ -662,7 +662,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
 - South Korea
   - [Busan](https://www.facebook.com/groups/free.code.camp.busan/)
   - [Daegu](https://www.facebook.com/groups/free.code.camp.your.daegu/)
-  - [Gwangju](https://www.facebook.com/groups/204942919854198/)
+  - [Gwangju](https://www.facebook.com/groups/GwangjuCodeCano/)
   - [Jeonju](https://www.facebook.com/groups/free.code.camp.jeonju/)
```

Comment:
sure it was m>n and p>o mistyped.

Sadly, sometimes I just receive their already defined link (which we cannot possibly physically enforce for them to get right before they do it themselves), it's already in a different convention —in spite of having clear suggestions in the guide...

And then FB in their infinite wisdom decided they won't let groups change the link once it's been defined. Between a rock and a hard place on this one.